scratch/evaluate_gaming_traffic.cc

```diff
@@ -72,7 +72,7 @@ void
 CalculateThroughput (Ptr<OutputStreamWrapper> stream)
 {
   Time now = Simulator::Now ();    /* Return the simulator's virtual time. */
-  double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / computeThroughputPeriodicity;     /* Convert Application RX Packets to kb. */
+  double cur = (packetSink->GetTotalRx () - lastTotalRx) * 8.0 / computeThroughputPeriodicity / 1e3;     /* Convert Application RX Packets to kb. */
```

Comment:
looks like `computeThroughputPeriodicity` is in [ms], you're assuming seconds here.
the best way to approach is to define it as a `Time` variable instead of a `double` variable and use `computeThroughputPeriodicity.GetSeconds ()` instead. It is more robust and easier to read

Reply:
Be defining a `Time` variable, how can I get a Time value from the command line?